renderers/team_renderer.py

```diff
@@ -27,12 +29,31 @@
 class TeamRenderer(object):
     @classmethod
     def render_team_details(cls, handler, team, year, is_canonical):
+        hof_award_future = award_query.TeamEventsTypeAwardsQuery(team.key.id(), EventType.CMP_FINALS, AwardType.CHAIRMANS).fetch_async()
+        hof_video_future = media_query.TeamTagMediasQuery(team.key.id(), MediaTag.CHAIRMANS_VIDEO).fetch_async()
+        hof_presentation_future = media_query.TeamTagMediasQuery(team.key.id(), MediaTag.CHAIRMANS_PRESENTATION).fetch_async()
+        hof_essay_future = media_query.TeamTagMediasQuery(team.key.id(), MediaTag.CHAIRMANS_ESSAY).fetch_async()
         media_future = media_query.TeamYearMediaQuery(team.key.id(), year).fetch_async()
         social_media_future = media_query.TeamSocialMediaQuery(team.key.id()).fetch_async()
-        robot_future = Robot.get_by_id_async('{}_{}'.format(team.key.id(), year))
+        robot_future = Robot.get_by_id_async("{}_{}".format(team.key.id(), year))
         team_districts_future = team_query.TeamDistrictsQuery(team.key.id()).fetch_async()
         participation_future = team_query.TeamParticipationQuery(team.key.id()).fetch_async()
 
+        hof_awards = hof_award_future.get_result()
+        hof_video = hof_video_future.get_result()
+        hof_presentation = hof_presentation_future.get_result()
+        hof_essay = hof_essay_future.get_result()
+
+        hall_of_fame = {
+            "is_hof": len(hof_awards) > 0,
+            "year": hof_awards[0].year if len(hof_awards) > 0 else None,
```

Comment:
191 won twice. This should handle that.
e.g. "Hall of Fame (1992, 1994)"

`"years": [award.year for award in hof_awards]`

Reply:
Should we handle multiple videos/presentations/essays? Doesn't apply here because those only started getting recorded in 2003, but _could_ apply in the future.

Reply:
Meh, unless you feel strongly about it/want to easily add it. I doubt another team will HOF repeat, so it's probably not a huge concern

Reply:
There also could technically be multiple videos of the presentation, but we can deal with it later if it ever happens.